samples/subsys/usb/uvc/boards/mini_stm32h743.conf

```diff
@@ -0,0 +1,8 @@
+# Switch from CDC ACM to UART
+CONFIG_USB_DEVICE_STACK=n
+CONFIG_USBD_CDC_ACM_CLASS=n
+CONFIG_UART_CONSOLE=y
+CONFIG_SERIAL=y
+
+# Avoid --- 1113 messages dropped --- when debugging
```

Comment:
I think this message can be more generic

Reply:
I removed the non-essential Kconfig options, allowing users to choose how they wish to tune their logging.